docs/getting-started/serving-simple-model.md

```diff
@@ -129,53 +160,50 @@ linear_regression
 └── src
     └── func_main.py
 ```
-@@@ note
+
+{% hint style="info" %}
 Although we have `model.py` inside the directory, it will not be uploaded to the cluster since we did not specify it in the contract's payload.
-@@@
+{% endhint %}
 
 ## Serving the model
 
-Now we can upload the model. Inside the `linear_regression` directory, execute the following command: 
+Now we can upload the model. Inside the `linear_regression` directory, execute the following command:
 
-```sh
+```bash
 $ hs -v upload
 ```
 
-Flag `-v` stands for verbose output. 
+Flag `-v` stands for verbose output.
 
-You can open the [http://localhost/models](http://localhost/models) page to see the uploaded model. 
+You can open the [http://localhost/models](http://localhost/models) page to see the uploaded model.
 
-Once you have opened it, you can create an __application__ for it. Basically, an application represents an endpoint to your model, so you can invoke it from anywhere. To learn more about advanced features, go to the [Applications](concepts/applications.md) page. 
+Once you have opened it, you can create an **application** for it. Basically, an application represents an endpoint to your model, so you can invoke it from anywhere. To learn more about advanced features, go to the [Applications](https://github.com/Hydrospheredata/hydro-serving/tree/54b7457851ad9de078cd092f083b8492dea6edca/docs/getting-started/concepts/applications.md) page.
```

Comment:
It seems that Gitbook doesn't delete page files, but rather removes pages from the documentation index structure to prevent them from showing. I've removed unused files to prevent further confusion.